cluster/clustered_service_agent.go

```diff
@@ -865,17 +881,37 @@ func (agent *ClusteredServiceAgent) getClientSession(id int64) (ClientSession, b
 	return session, ok
 }
 
-func (agent *ClusteredServiceAgent) closeClientSession(id int64) {
+func (agent *ClusteredServiceAgent) closeClientSession(id int64) bool {
 	if err := agent.checkForValidInvocation(); err != nil {
 		logger.Errorf("closeClientSession: error in checkForValidInvocation id=%d", id)
-		return
+		return false
```

Comment:
where do we handle java ClusteredException?

Reply:
Please see this PR which propagate all error to top for handling https://github.com/gravity-technologies/aeron-go/pull/31